modules/ROOT/assets/overlays/admin.yaml

```diff
@@ -0,0 +1,25 @@
+overlay: 1.0.0
+
+info:
+  title: Overlay for Admin REST API spec
+  version: 1.0.0
```

Comment:
Is this version going to show up in the display? The parent has 3.2 as the version https://github.com/couchbase/sync_gateway/blob/3.2-docs/docs/api/admin.yaml#L13

Reply:
It doesn't show up in the display. This is the version of the overlay, rather than the original spec. I don't want to tie it to the version of the spec because it would extra work to update the overlay files every time the spec updates, for no reason. This key is required by the OpenAPI overlay specification, otherwise I wouldn't have bothered to include it. I'll add a note to the overlay file explaining this.

Reply:
Done